packages/titus-backend/lib/plugins/healthcheck/index.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const fp = require('fastify-plugin')
+
+const { version } = require('../../../package.json')
+
+async function runCheck(server) {
+  let dbRes
+  try {
+    dbRes = await server.pg.query('SELECT $1::text as message', [
+      'Hello world!'
+    ])
+  } catch (err) {
+    // swallow error
+    server.log.debug({ err }, `failed to read DB during health check`)
+  }
+
+  return {
+    version,
+    serverTimestamp: new Date(),
+    status: 'ok',
+    memoryUsage: server.memoryUsage && server.memoryUsage(),
+    db: dbRes && dbRes.rowCount === 1 ? 'ok' : 'fail'
+  }
+}
+
+async function healthCheck(server, options, next) {
+  server.register(require('under-pressure'), parent => {
+    return {
+      ...options.underPressure,
+      healthCheck: async () => runCheck(parent)
+    }
+  })
+
+  next()
+}
+
+module.exports = fp(healthCheck, {
+  name: 'healthCheck',
+  dependencies: ['pg']
```

Comment:
this dependency doesn't look right. the name of the plugin is 'fastify-postgres'. please make sure you test this by running the app, this may pass the test but I think it breaks the app

Reply:
I'm requiring our own plugin, named `pg`

Reply:
I can require the generic one though

Reply:
ah sorry I missed that. I find it pointless to have that plugin but I understand now. it's fine